### DISCUSSION TITLE
Problem with setting up a project without TypeScript

### DISCUSSION THREAD

**Discussion by snhuynh** [keystonejs/keystone#7535] [Questions] [answered]
I've been trying to setup Keystone 6 for a project, but it's unclear to me how to setup it up _without_ typescript. Although the docs point out the steps to do so, I'm getting the following message after exactly following the steps mentioned in [this](https://keystonejs.com/docs/walkthroughs/lesson-1) guide from the docs:
```
It looks like you're trying to use TypeScript but do not have the required package(s) installed.

Please install typescript by running:

        npm install --save-dev typescript

If you are not trying to use TypeScript, please remove the tsconfig.json file from your package root (and any TypeScript files in your pages directory).

error Command failed with exit code 1.
```
This happens after running `yarn keystone dev`. I can confirm there is no usage of TS inside of my project.

Any one who's managed to get this up and running without typescript? The `create-keystone-app` utility hasn't given me the choice to not use TS so I was hoping a manual approach would do the trick.

**Context:**
Node: 16.15.0
Yarn: 1.22.18
OS: Arch Linux

**Comment by mattkrins** [comment]
I ran into this issue too. I just installed typescript anyway as it suggested and it seems to work even though i am not using any typescript in the project (that I'm aware of). I do though get lots of IDE errors which i believe is resulting from this inconsistency.

**Comment by java-jive** [comment]
This is similar to my issue, I installed `yarn add --dev @types/react` but it still complaining about it.
I tried also `typescript @types/node @types/react @types/react-dom @types/jest` but still no luck.

![image](https://user-images.githubusercontent.com/107587246/173999489-91c5325e-1788-490b-974a-d5224f3562fd.png)

**Comment by Achisingh** [comment, marked as answer]
@zeepsopje  confirming we want to fix this. I've created an issue here: https://github.com/keystonejs/keystone/issues/7636.
